### ISSUE TITLE
How do you even use this

### ISSUE THREAD

**Issue by RogueYin** [teralove/player-kills#1]
How do you check the kill count exactly?

**Comment by Zezo12345** [comment]
When you login proxy will say the number of kills for you, i have 4915 so far :)

**Comment by RogueYin** [comment]
Hm, guess its not working for my character despite it being super old with plenty of past battles outside lumber, ah well.

**Comment by Zezo12345** [comment]
Yes on my warrior, it works from time to time after a few relogs, but the other 5 charc that i tested on it worked from first time.

**Comment by Zezo12345** [comment]
My warrior had most of its kills in lumbertown/crescentia. So maybe its glitchy in this regard. Although it did show the kills after a few relogs when i noticed it wasnt showing on my warrior.
